dataland-frontend/tests/e2e/fixtures/p2p/P2pPreparedFixtures.ts

```diff
@@ -5,24 +5,24 @@ import { generateP2pFixtures } from "./P2pDataFixtures";
 /**
  * Generates Pathway To Paris prepared fixtures by generating random Pathway To Paris datasets and afterwards manipulating some fields
  * via manipulator-functions to set specific values for those fields.
- * @param undef_probability probability for a field to be undefined
+ * @param undefinedProbability probability for a field to be undefined
  * @param toggleRandomSectors determines if the sector list should include all possible sectors or a randomized selection
  * @returns the prepared fixtures
  */
 export function generateP2pPreparedFixtures(
-  undef_probability: number,
+  undefinedProbability: number,
```

Comment:
This should rather be called nullProbability instead of undefinedProbability

Reply:
Done